clerk/organizations.go

```diff
@@ -161,3 +161,60 @@ func (s *OrganizationsService) CreateInvitation(params CreateOrganizationInvitat
 	_, err := s.client.Do(req, &organizationInvitation)
 	return &organizationInvitation, err
 }
+
+type ListMembershipsParams struct {
+	OrganizationID string
+	Limit          *int
+	Offset         *int
+}
+
+type publicUserData struct {
+	FirstName       *string `json:"first_name"`
+	LastName        *string `json:"last_name"`
+	ProfileImageURL string  `json:"profile_image_url"`
+	Identifier      string  `json:"identifier"`
+}
+
+type organizationPublicUserData struct {
+	publicUserData
+	UserID string `json:"user_id"`
+}
+
+type OrganizationMembership struct {
+	Object          string          `json:"object"`
+	ID              string          `json:"id"`
+	PublicMetadata  json.RawMessage `json:"public_metadata"`
+	PrivateMetadata json.RawMessage `json:"private_metadata,omitempty"`
```

Comment:
:wrench: I think we will always have `PrivateMetadata` here, it's ok not to `omitempty`.